ddtrace/contrib/kafka/patch.py

```diff
@@ -33,6 +33,10 @@
 
 
 class TracedProducer(confluent_kafka.Producer):
+    def __init__(self, config, *args, **kwargs):
+        super(TracedProducer, self).__init__(config, *args, **kwargs)
+        self._dd_bootstrap_servers = config.get("bootstrap.servers") or config.get("metadata.broker.list") or None
```

Comment:
Do we expect all python statements which evaluate to `False` to be remapped to None?  For instance `[]`?

Reply:
Turns out my recollection of crashes of confluent_kafka upon giving it non-string values was unfounded. I'll make this separate and explicitly not reliant on falsey values.